src/main/groovy/edu/oregonstate/mist/locations/mapper/LocationMapper.groovy

```diff
@@ -26,13 +26,17 @@ class LocationMapper  {
             geoLocation: createGeoLocation(campusMapLocation.latitude,
                                             campusMapLocation.longitude),
             geometry: new Geometry(
-                coordinates: campusMapLocation.coordinates),
+                coordinates: campusMapLocation.coordinates,
+                type: campusMapLocation.coordinatesType),
             address: campusMapLocation.address,
             summary: campusMapLocation.shortDescription,
             description: campusMapLocation.description,
             thumbnails: [getImageUrl(campusMapLocation.thumbnail)] - null,
             images: [getImageUrl(campusMapLocation.largerImage)] - null,
-            type: Constants.TYPE_BUILDING
+            type: Constants.TYPE_BUILDING,
```

Comment:
Would a campusmap location have giRestroom data? Wouldn't the arcgis be the only datasource that comes with giRestroom data?